server.rb

```diff
@@ -74,14 +74,17 @@ def comment_created_message(redis, repository, issue, comment)
   end
 end
 
-def pull_request_opened_message(redis, repository, pull_request)
-  score = redis.get(pull_request.user.login).to_i
+def handle_labeled_event(event, redis, slack)
+  repository = event.repository
+  issue = event.issue
+  return unless event.labeled?
+  return unless repository.name =~ /site|internal-api|nightwatch|listener/;
```

Comment:
if we want to start filtering for some repositories then we should filter all the events, not just the labeled ones